doc/api/async_hooks.md

```diff
@@ -170,6 +170,13 @@ provided by AsyncHooks itself. The logging should then be skipped when
 it was the logging itself that caused AsyncHooks callback to call. By
 doing this the otherwise infinite recursion is broken.
 
+### Class: `AsyncHook`
+
+The class `AsyncHook` exposes interface for tracking lifetime events
+of asynchronous operations. Using a unique ID per asynchronous call,
+it helps associating method calls that otherwise do not appear together
```

Comment:
Either 

```suggestion
it helps with associating method calls that otherwise do not appear together
```

or 

```suggestion
it helps associate method calls that otherwise do not appear together
```

That being said – what’s a typical call stack?

Reply:
@addaleax , I used `typical` here to mean a representative call stack on node.js program error. As per my understanding, a representative stack is a incomplete part of an asynchronous sequence. Am I wrong?

Reply:
@HarshithaKP I’m still confused… That kind of only moves the question from “what’s a typical call stack?” to “what’s a representative call stack?”.

To be honest, I think it’s okay to drop the second sentence here. The sentence is not accurate anyway, the IDs are unique per asynchronous resource but not per asynchronous call.

Reply:
@addaleax, dropped the second sentence.